c_src/ep_decoder.c

```diff
@@ -591,6 +629,22 @@ unpack_field(ErlNifEnv *env, ep_dec_t *dec, ERL_NIF_TERM *term, wire_type_e wire
     ERL_NIF_TERM    ret;
 
     *term = 0;
+
+    if (field->ebin == TRUE) {
+        switch (field->type) {
+        // currently, we only support int64.
```

Comment:
Why add this code here and not just add it down in the right case for i64?

Reply:
fixed.